engine/apps/api/serializers/channel_filter.py

```diff
@@ -143,7 +143,7 @@ def get_filtering_term_as_jinja2(self, obj):
 
 
 class ChannelFilterCreateSerializer(ChannelFilterSerializer):
-    slack_channel_id = SlackChannelsFilteredByOrganizationSlackWorkspaceField(
```

Comment:
had accidentally changed this from `slack_channel` to `slack_channel_id`. The UI currently sends this field as `slack_channel` (inconsistent with how the Schedules endpoint expect this field.. should probably refactor/make-consistent at some point)